python_modules/dagster/dagster/_core/definitions/declarative_scheduling/asset_condition.py

```diff
@@ -411,6 +411,13 @@ def all_deps_match(condition: "AssetCondition") -> "AssetCondition":
 
         return AllDepsCondition(operand=condition)
 
+    @staticmethod
+    def missing_() -> "AssetCondition":
```

Comment:
why the trailing underscore?

Reply:
This is just a temporary state of affairs -- there's already an existing `missing` static constructor on `AssetCondition`, which routes to a legacy RuleCondition.

Once AssetCondition gets broken into its own separate thing later on, I'm able to name this just `missing()`